tests/zfs-tests/include/default.cfg.in

```diff
@@ -192,6 +192,15 @@ if is_linux; then
 	DEV_MPATHDIR="/dev/mapper"
 
 	NEWFS_DEFAULT_FS="ext2"
+
+	if [[ -e /var/log/messages ]]; then
+		export ZED_LOG="/var/log/messages"
+	elif [[ -e /var/log/syslog ]]; then
+		 export ZED_LOG="/var/log/syslog"
+	else
+		log_note "System logging path not found"
+		export ZED_LOG=""
+	fi
```

Comment:
@behlendorf are you suggesting not using log at all? I was thinking of specifying the zedlet directory when starting zed like you suggested, and before that copying "/usr/local/etc/zfs/zed.d/all-syslog.sh" into either /var/tmp or some other directory (to bypass in-tree permissions of zedlets) and then still parsing the syslog for the events (more filtered then zpool events).  Unless you were trying to suggest something else?